app/deploy/README.md

```diff
@@ -108,23 +103,25 @@ OpenShift includes the ability for a service account to act as a limited OAuthCl
 for more details). Let's create the service account with the correct redirect URIs enabled:
 
 ```bash
-$ oc create -f https://raw.githubusercontent.com/syndesisio/syndesis-openshift-templates/master/support/serviceaccount-as-oauthclient-restricted.yml
+$ oc create -f support/serviceaccount-as-oauthclient-restricted.yml
 ```
 
 #### Create the template to use
 
-We will create the template in the project, rather than in the openshift namespace as it is assumed the user does not have cluster-admin rights:
+Create the template:
 
 ```bash
-$ oc create -f https://raw.githubusercontent.com/syndesisio/syndesis-openshift-templates/master/syndesis-dev-restricted.yml
+$ oc create -f syndesis.yml
```

Comment:
Should this point to raw.githubusercontent.com...?

Reply:
It refers to local file in git repo, imho.

Reply:
Yes, its just meant as an example. wanted to get rid of the URL which is bogus anyway and I just assume that you cd is the one where the README is.

And tbh, we should maybe even remove this doc here and _only_ reference to the manual.